docs/modules/reference/pages/daemons/introduction.adoc

```diff
@@ -39,8 +39,8 @@ See the xref:operation:deep-dive/admin/daemon-config-files.adoc[system administr
 | Discovery
 | Provides network discovery functions to scan (either one time only or regularly) for nodes that should be added into the {page-component-title} database.
 
-| EnLinkd
-| The enhanced link daemon discovers connections between nodes using data generated by various link discovery protocols and accessible via SNMP.
+| Enlinkd
+| Discovers connections between nodes using data generated by various link discovery protocols, and that are accessible via SNMP.
```

Comment:
Please update based on solution to question about this phrasing at start of this PR.

Reply:
```suggestion
| Discovers connections between nodes using data generated by various link discovery protocols.
```